gammapy/time/plot.py

```diff
@@ -2,31 +2,126 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-__all__ = ['plot_time_difference_distribution',
+from ..datasets import fetch_fermi_catalog
```

Comment:
This fails on travis-ci:
https://travis-ci.org/gammapy/gammapy/jobs/66665216#L1200
It should fail locally for you too if you run
```
python setup.py test -P time
```
I think the issue is circular imports (see #171) and the short-term solution is to move the import into the `plot_light_curve` function.